ExpressFirstApp/index.js

```diff
@@ -16,6 +18,68 @@ app.use(express.static(staticDirectory));
 app.use(accessLogger);
 app.use(durationLogger);
 app.use(corsMiddleware);
+app.use(bodyParser.json());
+
+//Creating new todo files or appending to already created
+app.post('/todo', (req, res) => {
+  const { todoName, id, task } = req.body;
+  const log = `${id}. ${task} \n`;
+
+  fs.appendFile(`./todo/${todoName}.txt`, log, (err) => {
+    if (err) {
+      throw err;
+    }
```

Comment:
It will be good to have response to the client with status of the executed action.
Console log is always good to have, for example when throubleshoot the application.